api/room/mutations.js

```diff
@@ -39,4 +42,16 @@ export default {
       return { success: false, message: "Failed to add user" };
     }
   },
+  showHideRoomCard: async (_, { roomId, isShown }) => {
```

Comment:
I'd recommend using the updateRoom mutator instead of creating one specific to show/hide room

Reply:
Changed to use updateRoom mutation